addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Update minimum ZAP version to 2.14.0.
 - Update references for Expression Language Injection and HTTP Parameter Pollution (Issue 8262).
+  - Exponential Entity Expansions 
```

Comment:
Combine this with the previous one.

Reply:
There's really no previous one, this was already released.

Reply:
I meant the line right above. Or is this missing a rebase?

Reply:
Missing a rebase, this is no longer part of the Unreleased section.

Reply:
so i don't need to make changes here ?

Reply:
You need to rebase first and then move "this" (with changes) to the Unreleased section.